polls/admin.py

```diff
@@ -30,7 +33,34 @@ def queryset(self, request, queryset):
         return queryset
 
 
+def download_as_csv(QuestionAdmin, request, queryset):
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment;filename=export.csv'
```

Comment:
Could we make the filename more descriptive? When someone has multiple downloads, they would get confused about which export is which.

Maybe it should contain the export date and the fact that it is a question export? Something like `tuneme-questions-2016-01-11.csv` or something?